src/app/clients/logging-client/logging-client.ts

```diff
@@ -8,7 +8,8 @@ export const LoggingClient = {
   }: {
     userId: string;
     logs: Array<{
-      date: Date;
+      createdAt: Date;
+      name: String | Date;
```

Comment:
```suggestion
      name: string | Date;
```
I was also wondering what your thoughts were about using a string and a date for the name?